### DISCUSSION TITLE
PyBullet Code

### DISCUSSION THREAD

**Discussion by mhansen12** [community/community#45210] [Other Feature Feedback, Questions, & Ideas]
Hi all, I am new to pybullet, normally I work with matlab/simscape but I want to switch to pybullet. I have a few questions.
1) Is there a joint with 6DoF? I haven't found one. In Matlab you can use the bushing joint for that.  In a URDF it is described as a floating joint.
2) Is it possible that each degree of freedom has its own stiffness and damping parameters? and if so, how can I model that?
3) Can I apply a force to the tool centre in each direction of the degree of freedom? and if so, how can I do this and how can I determine the resulting displacement? 

thank you for your help

**Comment by queenofcorgis** [comment]
Hi @mhansen12 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this, such as https://pybullet.org/. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
